### PR TITLE
cloudflared 2022.2.2 (new formula)

### PR DESCRIPTION
**Pull request by piperswe**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/cloudflared.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  depends_on "gettext" => :build
+  depends_on "go" => :build
+
+  def install
+    system "make", "cloudflared"
+    system "sh", "-c",
+      'DATE="\\"$(date -u \'+%Y-%m-%d-%H%M UTC\')\\"" '\
+      'VERSION="$pkgver" '\
+      "envsubst < cloudflared_man_template > cloudflared.1"
```
Comment (carlocab) [inline Formula/cloudflared.rb:17]: Can we use `with_env` and the `Formula` `time` method here? It'll save an `sh` call (which is more efficient), and will use a date that doesn't depend on the exact time the build runs (which is more reproducible).

Comment (SMillerDev) [reply Formula/cloudflared.rb:17]: Should be like this, but I'm typing this in the web editor
```suggestion
    with_env(DATE: time.iso8601, VERSION: version) do
      system  "envsubst < cloudflared_man_template > cloudflared.1"
    end
```

Comment (piperswe) [reply Formula/cloudflared.rb]: Thanks for the tip!

**Review by carlocab — APPROVED** [review]
Thanks; this looks so much nicer.

**Comment by chenrui333** [comment]
@piperswe, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**